rialto_airflow/harvest/doi_sunet.py

```diff
@@ -10,7 +10,7 @@
 
 def create_doi_sunet_pickle(
     dimensions: str, openalex: str, sul_pub_csv: str, authors_csv: str, output_path
-) -> dict:
+) -> None:
```

Comment:
doesn't actually seem to return anything

Reply:
Yes, I think these pickles are going to go in the dustbin, so it doesn't matter too much I don't think.